internal/generator/vector/inputs.go

```diff
@@ -153,6 +153,22 @@ func Inputs(spec *logging.ClusterLogForwarderSpec, o Options) []Element {
 		}
 	}
 
+	for _, input := range spec.Inputs {
+		if input.Source != nil && input.Source.HTTP != nil && input.Source.HTTP.LogType == `audit` {
```

Comment:
instead of "audit" we should the well known types from the api dir

Reply:
+1 you mean the constant `loggingv1.InputTypeAudit` right?